Finish Junit Test Except RestWebController

Maysaraodeh opened the pull request:
there is a problem with restWebController getObjects Method.

Commit:
finish Junit test except RestWebController

the junit test has finished except the issue is that the target object
is null in the RestWebController getObjects method

Commit:
Fixed CSRF. Added tags logic and only styling left.

Commit:
Forgot to include in last commit.

sariitg commented on pom.xml:
```diff
@@ -30,6 +30,11 @@
 		</dependency>
 		<dependency>
+    		<groupId>com.google.code.gson</groupId>
+    		<artifactId>gson</artifactId>
```
Don't forget indentation

sariitg replied:
test

sariitg commented on src/main/java/com/paliup/nutrition/config/AppConfig.java:
```diff
@@ -1,15 +1,32 @@
+
+	@Bean
+	public HomeController transferServiceHomeController() {
+		return new HomeController();
```
Fails to run server

sariitg commented on src/main/java/com/paliup/nutrition/service/CustomUserDetails.java:
```diff
@@ -20,6 +21,8 @@ public CustomUserDetails(User user, List<String> userRoles) {
 		this.userRoles = userRoles;
 	}
 
+	public CustomUserDetails() {}
```
Double check if you need this.

sariitg commented on src/test/java/com/paliup/nutrition/config/AppConfigTest.java:
```diff
@@ -0,0 +1,39 @@
+	@Test
+	public void testPersistanceServiceBean() {
+		assertEquals("class com.paliup.nutrition.service.PersistanceService",
+				this.appConfig.transferService().getClass().toString());
```
You don't need the "this"

sariitg replied:
You still dont need "this" and there are unused imports

sariitg commented on src/test/java/com/paliup/nutrition/controller/HomeControllerTest.java:
```diff
@@ -0,0 +1,44 @@
+	HomeController controller = new HomeController();
+
+	// @Autowired
+	// private HomeController controller;
```
Either autowire or instantiate. Remove the unwanted one.

sariitg commented on src/test/java/com/paliup/nutrition/service/CustomUserDetailsTest.java:
```diff
@@ -2,27 +2,68 @@
+	@Test
+	public void testIsAccountNonLocked() {
+		customUserDetails.isAccountNonLocked();
+		assertTrue(true);
```
assertTrue(customUserDetails.isAccountNonLocked());

sariitg commented on src/test/java/com/paliup/nutrition/service/CustomUserDetailsTest.java:
```diff
@@ -2,27 +2,68 @@
+	@Test
+	public void testIsCredentialsNonExpired() {
+		customUserDetails.isCredentialsNonExpired();
+		assertTrue(true);
```
^^

sariitg commented on src/test/java/com/paliup/nutrition/service/CustomUserDetailsTest.java:
```diff
@@ -2,27 +2,68 @@
+	@Test
+	public void testIsEnabled() {
+		customUserDetails.isEnabled();
+		assertTrue(true);
```
^^

sariitg commented on src/test/java/com/paliup/nutrition/service/CustomUserDetailsTest.java:
```diff
@@ -2,27 +2,68 @@
+	}
+
+	@Test
+	public void tesGetUsernameShouldReturnUserName() {
```
Ignore this test

sariitg commented on src/test/java/com/paliup/nutrition/service/CustomUserDetailsTest.java:
```diff
@@ -2,27 +2,68 @@
+	}
+
+	@Test
+	public void testGetAuthoritiesShouldReturnAllAuthorities() {
```
^^

Commit:
fix some issues with Junit test

fix restWebController issues
fix CustomerUserDetails issues
fix indentation
and others

Commit:
fix project error

fix the error eith the HomeController Bean

Commit:
fix indentation again

Commit:
fix indentation

Commit:
Quick updates to Maysara commit

Commit:
Added phone number
